modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/util/StringUtil.java

```diff
@@ -5,9 +5,14 @@
 
 package com.liferay.gradle.plugins.workspace.internal.util;
 
+import com.liferay.petra.string.CharPool;
```

Comment:
Sorry I missed this before, @thiago-buarqque, but I'd like to keep all `petra` dependencies out of workspace for now. You can add the colon as a constant field directly here in `StringUtil` (there are a few others like it).

Reply:
@drewbrokke Done